integration_test/third_party_apps_data/agent/ops-agent/linux/enable_couchdb

```diff
@@ -0,0 +1,36 @@
+# Configures Ops Agent to collect telemetry from the app and restart Ops Agent.
+
+set -e
+
+sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << EOF
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+metrics:
+  receivers:
+    couchdb:
+      type: couchdb
+      endpoint: http://localhost:5984
```

Comment:
Please test the default values for these parameters (so no endpoint, collection interval, username, password, etc.)

Reply:
Hey Quentin, the default configuration will not work. In order to curl the metrics endpoint, a user:password is required or else an `unauthorized` error is returned. Since the default admin user has no default password a [new admin](https://docs.couchdb.org/en/stable/intro/security.html#creating-a-new-admin-user) needs to be created which can be done in the local.ini file.

This also means that the enable_couchdb config will require a username and password too.

Reply:
Thanks for the clarification. Looks like all parameters have been removed except username and password, which as explained are required.

Resolving this.